Sources/StreamChat/Workers/ReadStateSender.swift

```diff
@@ -4,74 +4,96 @@
 
 import Foundation
 
-@available(iOS 13.0, *)
-extension Chat {
-    actor ReadStateSender {
-        private let cid: ChannelId
-        private let authenticationRepository: AuthenticationRepository
-        private let messageRepository: MessageRepository
-        private let channelUpdater: ChannelUpdater
-        
-        init(
-            cid: ChannelId,
-            channelUpdater: ChannelUpdater,
-            authenticationRepository: AuthenticationRepository,
-            messageRepository: MessageRepository
-        ) {
-            self.authenticationRepository = authenticationRepository
-            self.cid = cid
-            self.channelUpdater = channelUpdater
-            self.messageRepository = messageRepository
+final class ReadStateSender {
```

Comment:
I would rename it to something like this:
```suggestion
final class ReadStateHandler {
```

The "Sender" postfix is used as a background worker, like the "MessageSender", so it is used fro components which are observing data changes and then queue and send requests async. 

This component the goal is just to reuse the State logic, kinda like we have the PaginationStateHandler. So I guess it makes sense to have consistancy and also have the "Handler" postfix

Reply:
Sure, let's do that. I'll do another commit and fix all the call sites.